modules/101-cert-manager/docs/FAQ.md

```diff
@@ -176,3 +176,67 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
     - www.example.com
   EOF
   ```
+
+## How to secure cert-manager credentials
+
+If you don't want to store credentials in the Deckhouse configmap (security reasons for example), feel free to create
+your own ClusterIssuer / Issuer.
+For example, you can create your own ClusterIssuer for route53 in this way:
+1. Create secret with credentials:
```

Comment:
```suggestion
1. Create a Secret with credentials:
```

Reply:
Also, please set indent of code blocks to 3 or there will be a space on the left after rendering.